CMakeLists.txt

```diff
@@ -212,33 +206,7 @@ install(FILES ${ArrayFire_BINARY_DIR}/include/af/version.h
         DESTINATION "${AF_INSTALL_INC_DIR}/af/"
         COMPONENT headers)
 
-if(Forge_FOUND AND NOT AF_USE_SYSTEM_FORGE)
-    option(AF_INSTALL_FORGE_DEV "Install Forge Header and Share Files with ArrayFire" OFF)
-    mark_as_advanced(AF_INSTALL_FORGE_DEV)
-    af_deprecate(INSTALL_FORGE_DEV AF_INSTALL_FORGE_DEV)
-    if(AF_INSTALL_FORGE_DEV)
-        install(DIRECTORY "${ArrayFire_BINARY_DIR}/third_party/forge/include/"
-                DESTINATION "${AF_INSTALL_INC_DIR}"
-                COMPONENT headers
-        )
-        install(DIRECTORY "${ArrayFire_BINARY_DIR}/third_party/forge/share/Forge/"
-                DESTINATION "${AF_INSTALL_DATA_DIR}/../Forge"
-                COMPONENT share
-        )
-        install(DIRECTORY "${ArrayFire_BINARY_DIR}/third_party/forge/lib/"
-                DESTINATION "${AF_INSTALL_LIB_DIR}"
-                COMPONENT forge
-        )
-    endif()
-    #install forge library & dependencies
-    set(fg_dlib_px "bin")
-    if (UNIX)
-      set(fg_dlib_px "lib")
-    endif ()
-    install(DIRECTORY "${PROJECT_BINARY_DIR}/third_party/forge/${fg_dlib_px}/"
-      DESTINATION "${AF_INSTALL_LIB_DIR}"
-      COMPONENT common_backend_dependencies)
-endif()
+#TODO(pradeep) install forge dependency for packaging - not required for builds
```

Comment:
stale TODO?

Reply:
nope, I am not sure how cpack stuff for graphics deps will be handled. So added this TODO to make sure we don't forget it.

Reply:
I wonder if that sort of thing should be added to cpackconfig.cmake